### PR TITLE
Add taskImageLink

### PR DESCRIPTION
**Pull request by Razzmatazzz**
Also bumps deps.

### CODE REVIEW

**Comment by Razzmatazzz** [comment]
.deploy

File: .github/workflows/branch-deploy.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: github/branch-deploy@v6.0.0
+      - uses: github/branch-deploy@v6.0.1
```
Comment (GrantBirki) [inline .github/workflows/branch-deploy.yml:21]:
```suggestion
      - uses: github/branch-deploy@v7.0.0
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       # https://github.com/github/branch-deploy/blob/d3c24bd92505e623615b75ffdfac5ed5259adbdb/docs/merge-commit-strategy.md
       - name: deployment check
-        uses: github/branch-deploy@v6.0.0
+        uses: github/branch-deploy@v6.0.1
```
Comment (GrantBirki) [inline .github/workflows/deploy.yml:20]:
```suggestion
        uses: github/branch-deploy@v7.0.0
```

File: .github/workflows/unlock-on-merge.yml [hunk 3]
```diff
@@ -16,7 +16,7 @@ jobs:
 
     steps:
       - name: unlock on merge
-        uses: github/branch-deploy@v6.0.0
+        uses: github/branch-deploy@v6.0.1
```
Comment (GrantBirki) [inline .github/workflows/unlock-on-merge.yml:19]:
```suggestion
        uses: github/branch-deploy@v7.0.0
```

**Review by GrantBirki — APPROVED** [review]

**Comment by GrantBirki** [comment]
.deploy

**Pull request merged**